util/duplicate_detector.h

```diff
@@ -5,8 +5,7 @@
 
 #pragma once
 
-#include <cinttypes>
-
+#include "db/db_impl/db_impl.h"
```

Comment:
Not disagreeing with this, but seems like a big hammer for something in an include file....

Reply:
I'm just including what is needed, and clearly needed in this case. (`db_->immutable_db_options()` etc.)

Reply:
Agreed, just a big hammer :(